src/main/java/com/mentoring/ecommerce/application/service/ProductSaveService.java

```diff
@@ -0,0 +1,19 @@
+package com.mentoring.ecommerce.application.service;
+
+
+import com.mentoring.ecommerce.application.port.in.SaveProductUseCase;
+import com.mentoring.ecommerce.application.port.out.SaveProductPort;
+import com.mentoring.ecommerce.domain.Product;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class ProductSaveService implements SaveProductUseCase {
+
+    private SaveProductPort getProduct;
```

Comment:
@Pelicer, why doesn't the `Service` and `Controller` use `@Autowired` annotation in their attributes?